### PR TITLE
Convert objToJSON methods to HPy

### PR DESCRIPTION
**Pull request by hodgestar**

### CODE REVIEW

File: python/objToJSON.c [hunk 1]
```diff
@@ -844,70 +882,90 @@ PyObject* objToJSON(PyObject* self, PyObject *args, PyObject *kwargs)
...
 
-  if (!PyArg_ParseTuple (args, "OO", &data, &file))
+  if (!HPyArg_Parse(ctx, &ht, args, nargs, "OO", &data, &file))
   {
-    return NULL;
+    return HPy_NULL;
   }
 
-  if (!PyObject_HasAttrString (file, "write"))
+  file_o = HPy_AsPyObject(ctx, file);
```
Comment (fangerer) [inline python/objToJSON.c]: Is it really necessary to convert to `PyObject *` here? I think we do already have all necessary API functions to operate on handles.

File: python/ujson.c [hunk 2]
```diff
@@ -39,40 +39,35 @@ Numeric decoder derived from from TCL library
...
     &JSONFileToObj,
+    &objToJSON,
+    &objToJSON_encode,
+    &objToJSONFile,
     NULL
 };
 
-#define ENCODER_HELP_TEXT "Use ensure_ascii=false to output UTF-8. Pass in double_precision to alter the maximum digit precision of doubles. Set encode_html_chars=True to encode < > & as unicode escape sequences. Set escape_forward_slashes=False to prevent escaping / characters."
-
-static PyMethodDef ujsonMethods[] = {
```
Comment (fangerer) [inline python/ujson.c:62]: woohoo 🥳

**Review by fangerer — APPROVED** [review]
I would appreciate if we could get rid of calls to `HPy_AsPyObject` but that's no blocker for me and we can do it later.

File: python/objToJSON.c (hunk 1 above)
Comment (hodgestar) [reply python/objToJSON.c]: Good idea! Done. :D

HPy has come a long way since we first worked on the ultrajson port!

**Review by antocuni — APPROVED** [review]
LGTM

**Pull request merged**